tests/notifier/test_keep_alive_monitor.py

```diff
@@ -68,20 +69,18 @@ def callback(events: List[Event]):
 
         begin_tp = datetime.now()
 
-        for _ in range(self.threshold_seconds):
-            self.keep_alive_monitor.process_events(self.keep_alive_events)
-            sleep(1)
+        with vcr.use_cassette("tests/cassette/keep_alive_monitor_remote_ping.yaml", record_mode="none") as cass:
+            for _ in range(self.threshold_seconds):
+                self.keep_alive_monitor.process_events(self.keep_alive_events)
+                sleep(1)
 
-        while not received_high_priority_event:
-            logging.info(f"Waiting for high priority event, this should only take {self.threshold_seconds}s")
-            sleep(1)
+            while not received_high_priority_event:
+                logging.info(f"Waiting for high priority event, this should only take {self.threshold_seconds}s")
+                sleep(1)
+            assert cass.play_count == 2, "Remote ping request did not happen as expected."
 
         end_tp = datetime.now()
         seconds_elapsed = (end_tp - begin_tp).seconds
 
         # Check that high priority event did not fire before keep-alive signal stopped
         self.assertGreater(seconds_elapsed, 2 * self.threshold_seconds - 1)
-
-
-if __name__ == "__main__":
-    unittest.main()
```

Comment:
I personally like the additional convenience of being able to run individual test file as main.

Reply:
It saps coverage since it's never run, but they could be kept with a pragma no cover if you prefer?

Reply:
Ah I see